nservicebus/operations/async-analyzer.md

```diff
@@ -19,9 +19,9 @@ The analyzer will point out missing `await` operators when using asynchronous me
 
 ## Limitations
 
-This analyzer inspects only the APIs contained on the classes and interfaces as described in the previous section. Other asynchronous operations on NServiceBus APIs are not analyzed.
+This analyzer inspects only the APIs contained on the classes and interfaces as described in the previous section. Other asynchronous methods on NServiceBus APIs are not analyzed.
 
-The analyzer checks for ignored `Task`s returned from the scanned methods. If the `Task` is assigned to a variable or passed to another method, the analyzer will not analyze this `Task`'s usage any further.
+The analyzer checks for ignored `Task`s returned from the scanned methods. If the `Task` is assigned to a variable, passed to another method or one if its members are accessed, the analyzer will not analyze this `Task`'s usage any further.
```

Comment:
Second revision:

"If the `Task` is assigned to a variable, passed to another method, or has one of its members accessed, the analyzer will not evaluate this `Task`'s usage any further."